vulcanai/tests/models/test_metrics.py

```diff
@@ -330,3 +330,12 @@ def test_run_test_binary(self, metrics, cnn_class_binary):
                             'macro_npv', 'macro_f1', 'macro_auc']
 
         assert all(k in res_dict for k in required_metrics)
+
+    def test_sensitivity_analysis(self, metrics, dnn_class_single_value_2,
+                                  sensitivity_data_loader):
+        metrics.conduct_sensitivity_analysis(dnn_class_single_value_2,
```

Comment:
perhaps you can look at the saved file that this function leaves after running to see if the expected outputs are correct. What we have here isn't necessarily enough. You can use the other test csv for it to be a more robust test

Reply:
Hm I'm not really sure how to do this... The network weights I have are random for the test

Reply:
It doesn't matter if the network weights are random, as long as you initialize them exactly the same way each time (hint: setting the seed) then you can have deterministic output of the model because your function will operate across the same values every-time in this scenario no? well then once it completes a test run, take a look at the file it saves, validate that it's been correctly calculated, and then hardcode the assertions to match exactly the values you find in the file. You can try to modify the function afterwards which would output a different value which would then fail your assertions. I hope laying it out like this makes sense

Reply:
Yeah it does, thanks for the explanation, I'll update the assertions to reflect this.